source/dubregistry/registry.d

```diff
@@ -225,7 +225,7 @@ class DubRegistry {
 		nfo["dateAdded"] = pack._id.timeStamp.toISOExtString();
 		nfo["owner"] = pack.owner.toString();
 		nfo["name"] = pack.name;
-		nfo["logoHash"] = pack.logoHash.rawData.toHexString;
+		nfo["logo"] = pack.logo.toString;
```

Comment:
this is getting really confusing now, can you rename the new field to logoID instead?